dora-metrics/pelorus
# fix: Document and automate release and versioning steps

## Linked Issues

resolves #874

## Description

Document and automate steps related to versioning/release.

## Testing Instructions

Run the new script with different options and check if they give the expected results.
